bedrock/newsletter/views.py

```diff
@@ -656,7 +656,7 @@ def newsletter_subscribe(request):
         # form error messages may contain unsanitized user input
         errors = [escape(e) for e in errors]
 
-        if request.is_ajax():
+        if request.headers.get("x-requested-with") == "XMLHttpRequest":
```

Comment:
Thinking out loud: should we write our own helper `is_ajax(request)` ? This is so far only the second time we've seen the change in the codebase, so may be not worth it

Reply:
I had a similar thought but passed on it since there's only 2 uses.